src/index.md

```diff
@@ -37,6 +37,7 @@ The ecosystem consist of various components:
 - [How to install the Mergin Maps plugin for QGIS](./setup/install-mergin-maps-plugin-for-qgis/)
 - [Working with Organisations](./setup/working-with-organisations/)
 - [Subscriptions and invoicing](./setup/subscriptions/)
+- [Use multiple servers in plugin](./setup/plugin-multi-server-use/) 
```

Comment:
The order is "Concepts, How-tos, References", can you move to how-to section.

Reply:
Also name "How to use multiple servers in Mergin Maps Plugin for QGIS"

Maybe we should move it to section "Manage Account & Project ", as this is not about install/sign up, but about signing in...

Reply:
for main menu , yuo need to add it to https://github.com/MerginMaps/docs/blob/main/src/.vuepress/sidebar/en.js